MIGRATION-v8.norg

```diff
@@ -56,7 +56,7 @@ version: 1.1.1
    - `sudo pacman -Syu luajit` or `lua51`: {https://github.com/nvim-neorg/neorg/issues/1342#issuecomment-2017814358}[comment]
    - `sudo apt install liblua5.1-0-dev`: {https://github.com/nvim-neorg/neorg/issues/1342#issuecomment-2017728318}[comment]
    - `brew install luarocks`: *not tested* {https://github.com/nvim-neorg/neorg/issues/1342#issuecomment-2020157539}[comment]
-   - NixOS: PR required
+   - NixOS: see {# NixOS}.
    -- I think work has been done to make a flake of luarocks.nvim+neorg but I might be wrong.
```

Comment:
Shall we delete this line?

I though your code was doing this lol

Reply:
No, my code is different